tests/high_performance_vm_test.go

```diff
@@ -0,0 +1,106 @@
+package tests
+
+import (
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/google/goexpect"
+	"kubevirt.io/kubevirt/pkg/kubecli"
+	"kubevirt.io/kubevirt/tests"
+)
```

Comment:
add `"kubevirt.io/kubevirt-ansible/tests"` so that you can see objects defined in `tests` package.